framework/doc/content/syntax/ICs/index.md

```diff
@@ -62,6 +62,13 @@ object's variable *must* be defined on the block - otherwise the IC for the
 next lowest block ID for the node is used - and so forth until one has the
 variable defined.
 
+## Old and Older ICs
+
+The ICs system supports the ability to set ICs on old and older states. This can be useful for setting initial velocity or acceleration conditions. It can be set with the [!param](/ICs/ConstantIC/state) parameter and specifying `OLD` or `OLDER`.
```

Comment:
```suggestion
The ICs system supports the ability to set ICs on old and older states. This can be useful for initializing old and older states of variables needed for various time integration schemes. It can be set with the [!param](/ICs/ConstantIC/state) parameter and specifying `OLD` or `OLDER`.
```